### ISSUE TITLE
For general Communication

### ISSUE THREAD

**Issue by rananna** [OpenSourceEBike/TSDZ2_wireless#107]
@4var1 ,  @casainho , There is no way to PM users on github. We have been using PR comments for this and this results in difficulty finding messages. On the github forum they recommend creating an issue to track communications, so here it is!

Anyway, my first message is that I have just completed tesiting the wireless remote for use as a release.
@casainho , It all test well and is ready for your ride tomorrow.

**Comment by rananna** [comment]
@casainho , @4var1 ,

For reference, here is what has been tried (and didn't work) for flashing the wireless controller with Filippods on the forum:
1. downgrade rev lev
2. upgrade rev level
3. optimized firmware ( -o3 set)  
4. released 0.6.2 firmware
5. latest firmware (release candidate)
6. latest wireless remote firmware (this DID work, suggesting he is flashing with nrf connect ok, and that something is different with the controller OTA update firmware, but I don't know what that would be....)
7. different 5v supply
8. a second board (he may have damaged two boards, but the bootloader TSDZ2_DFU appears in both cases strongly suggesting the boards are fine)

no errors seem to occur in nrf connect, however he does not see TSDZ2 in bluetooth after flashing. 
No leds light up in the latest firrmware

I tried reproducing this problem with a new dongle, but all is well, both the load of the bootloader followed by the OTA update work as expected.
Pretty hard to debug when you cannot see the problem!
Perhaps there some user error on his part, but I can't imagine what....
Very odd that the flash of the wireless remote worked well though.....
I told him to wait for a new release, but I expect he will have the same problem.
I am at a loss, do either of you have any ideas?

**Comment by casainho** [comment]
See the forum, seems that Windows instructions is missing the mass erase command!!

**Comment by stsdc** [comment]
There is possibility to open Discussion tab on any repo.

**Comment by rananna** [comment]
From forum results, it's pretty clear that openocd for windows has an outstanding mass_erase problem.
I will raise a new issue.

**Comment by 4var1** [comment]
> From forum results, it's pretty clear that openocd for windows has an outstanding mass_erase problem.
> I will raise a new issue.

I got a bit lost in all the replies - but doesn't everything work fine with 5v connected?

**Comment by 4var1** [comment]
@casainho i saw your reply about down assist looking red+green. I'm really surprised you don't see that as yellow. Unless it's different on the controller - to me it looks totally fine. If i blink fast or move my eyes you can see a red flash occasionally - but otherwise the mixing is good.

**Comment by rananna** [comment]
> > From forum results, it's pretty clear that openocd for windows has an outstanding mass_erase problem.
> > I will raise a new issue.
> 
> I got a bit lost in all the replies - but doesn't everything work fine with 5v connected?

Indeed. there was far too much back and forth on that topic. He never got it working with windows. The linux build was what gave him the successful flash. I asked him to update his stlink as I read on other forums that mass erase may not work with older firmware.

**Comment by rananna** [comment]
> @casainho i saw your reply about down assist looking red+green. I'm really surprised you don't see that as yellow. Unless it's different on the controller - to me it looks totally fine. If i blink fast or move my eyes you can see a red flash occasionally - but otherwise the mixing is good.

IMO, the PWM brightness has a big affect on mixing low brightness does not mix as well. However, I am not too fussed by the appearance. I find it OK, but this is definitely in the personal preference category.

**Comment by 4var1** [comment]
> > @casainho i saw your reply about down assist looking red+green. I'm really surprised you don't see that as yellow. Unless it's different on the controller - to me it looks totally fine. If i blink fast or move my eyes you can see a red flash occasionally - but otherwise the mixing is good.
> 
> IMO, the PWM brightness has a big affect on mixing low brightness does not mix as well. However, I am not too fussed by the appearance. I find it OK, but this is definitely in the personal preference category.

Yea - the only time i've had an issue was when I had a restricted view of the led - so one colour wasn't visible -then it really doesn't mix. That was slightly confusing - but I do think that when a lense/clear silicone or similar is placed over it it will look better too.

To me doesn't make sense that up/down result in the same response. I know it's marginal - but i think any extra positive feedback is always good...
